src/main/java/com/hpe/adm/octane/ideplugins/services/mywork/EvertonP1MyWorkService.java

```diff
@@ -124,6 +126,32 @@
                 .collect(Collectors.toList());
     }
 
+    @Override
+    public EntityModel getEntityFromUserItemIfNeeded(EntityModel entity) {
```

Comment:
I see similar code above? can't it be extracted & reused?

Reply:
It's what we've talked about. Those are methods from the services utility class and the problem is that we need one implementation for the old services and another implementation for the new service. That's why they were added into the interface implementing all the mywork services

Reply:
I don't know the topic u discussed, but seems so very wrong to me. I see a lot of duplicated code which shouldn't exist. Why not use an util class, parametrized with what you need for each release? (also, implementing myWorkServices for releases...really odd at a first glance)